src/runlib.rs

```diff
@@ -91,14 +126,29 @@ pub fn record_artifacts(
                     };
                     if symlink_metadata(&s_path)?.file_type().is_file() {
                         let (virtual_target_path, hashes) =
-                            record_artifact(&path, hash_algorithms)?;
+                            record_artifact(&path, hash_algorithms, lstrip_paths)?;
+                        if artifacts.contains_key(&virtual_target_path) {
+                            return Err(Error::VerificationFailure(format!(
```

Comment:
I like this check! I'm a little nervous about using VerificationError for this though. I wonder if we can extend the enum with something like `CaptureError` or `LinkGatheringError`. What do youuthink?

Reply:
Same idea! Actually, I added a `PrefixPathConflict` in first version, but deleted it for the sake of influence to fewer files.

I will add `LinkGatheringError` back soon.